client/main.lua

```diff
@@ -134,9 +134,9 @@ local function RegisterStashTarget()
 
     stashTargetBox:onPlayerInOut(function(isPointInside)
         if isPointInside and not entering and isOwned then
-            exports['qbx-core']:DrawText(Lang:t("target.open_stash"), 'left')
+            exports['qbx_core']:DrawText(Lang:t("target.open_stash"), 'left')
```

Comment:
This will still be broken. See proper placement here: https://qbox-project.github.io/converting#common-replacements